java/lp-service-spring-boot-3.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     ext {
         devopsGradleDir = "${rootDir}/oodle-devops/gradle"
 
-        springBootVersion = '3.+' // Version used for gradle plugin AND framework dependencies
+        springBootVersion = '[3.0.0,3.3.0]' // Version used for gradle plugin AND framework dependencies ; TODO [3.0.0,3.3.0] is added to avoid the latest version 3.4.0 is incompatible with some of our services. This needs to be reverted once we resolve the issues
```

Comment:
Should this be `[3.0.0,3.3.999]` ? Probably we want 3.3.6 or whatever, not to stop at 3.3.0

Reply:
Still, this is a genius solution! 💯

Reply:
cant hurt to do that